packages/nuxt/src/common/types.ts

```diff
@@ -5,93 +5,93 @@ export type SentryNuxtOptions = Omit<Parameters<typeof init>[0] & object, 'app'>
 
 type SourceMapsOptions = {
   /**
-   * Options for the Sentry Vite plugin to customize the source maps upload process.
+   * If this flag is `true`, and an auth token is detected, the Sentry integration will
```

Comment:
l (sorry for the bike shedding): Just was wondering if "integration" is a good term here? IIUC users interact with our nuxt module here, correct? So I'd go with module or just SDK. 

```suggestion
   * If this flag is `true`, and an auth token is detected, the Sentry integration will
```